cassdegrees/templates/list.html

```diff
@@ -59,7 +59,11 @@
                             {% if key != "id" %}
                                 <th>{{ key }}</th>
                             {% else %}
-                                <th>Select</th>
+                                <th>
+                                    <p class="text-center">
+                                        Select All<input title="Select All" type="checkbox" name="select_all" onclick="selectAll(this, '{{ title }}')">
```

Comment:
I fixed your problem.
All you need to a break. 
You also don't even need the paragraph tag
```
<th class="text-center">
     Select All </br>
     <input title="Select All" type="checkbox" name="select_all" onclick="selectAll(this, '{{ title }}')">
</th>
```
![image](https://user-images.githubusercontent.com/44992275/57504030-22283780-7336-11e9-86c3-94fea65b03c2.png)